.github/workflows/build.yml

```diff
@@ -76,6 +76,7 @@ jobs:
             $config=(Get-Content C:\ProgramData\docker\config\daemon.json | ConvertFrom-json)
           }
           $config | Add-Member -Force -Name "insecure-registries" -value @("$IPAddress/32") -MemberType NoteProperty
+          $config | Add-Member -Force -Name "allow-nondistributable-artifacts" -value @("$IPAddress/32") -MemberType NoteProperty
```

Comment:
IIUC not having this set was causing us to push invalid (missing base layers) Windows images. Would it be possible to update the documentation [here](https://github.com/buildpacks/lifecycle/blob/main/DEVELOPMENT.md#caveats) to inform contributors on Windows or our future selves to enable this setting locally?

Reply:
Done